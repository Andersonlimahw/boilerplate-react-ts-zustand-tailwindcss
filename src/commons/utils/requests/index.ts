export * from './api-config';
export * from './http-client';