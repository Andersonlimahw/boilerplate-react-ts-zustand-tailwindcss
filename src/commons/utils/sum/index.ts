interface SumInput {
  a: number;
  b: number;
}

export const sum = ( { a, b } : SumInput) => {
  return a + b;
}