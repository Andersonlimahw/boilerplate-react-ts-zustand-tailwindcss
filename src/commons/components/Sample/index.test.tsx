// REF: https://github.com/vitest-dev/vitest/blob/main/examples/react-testing-lib
import { describe, expect, it } from 'vitest';
import { render, screen } from '../../utils/tests';

import { Sample } from './';

describe('Sample', () => {
  it('Render', async () => {
    render(<Sample />);
    expect(screen.getByTestId('element-id')).toBeInTheDocument();
  })

  it('Snapshot', async () => {
    const { asFragment } = render(<Sample />);
    expect(asFragment()).toMatchSnapshot();
  })
})