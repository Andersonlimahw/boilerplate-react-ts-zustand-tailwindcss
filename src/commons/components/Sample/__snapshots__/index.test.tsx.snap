// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sample > Snapshot 1`] = `
<DocumentFragment>
  <h1
    data-testid="element-id"
  >
    Sample
  </h1>
</DocumentFragment>
`;
